scripts/retdec-config.py

```diff
@@ -87,5 +87,10 @@
 UNPACKER = os.path.join(INSTALL_BIN_DIR, 'retdec-unpacker')
 
 # Other.
-LOG_TIME = ['/usr/bin/time', '-v']
+
+if platform == "darwin":
+    # mac os x doesn't support the v flag
+    LOG_TIME = ['/usr/bin/time']
```

Comment:
Although this fixes the `/usr/bin/time: invalid option -- 'v'` error on macOS, it does not represent a full solution to #383 as our scripts assume that `/usr/bin/time` is run with `-v`. Apart from other information, the `-v` parameter makes `/usr/bin/time` produce memory usage of the tracked program, which we then parse in our scripts. Without `-v`, the used memory will always be 0, which I believe is incorrect and thus this fix cannot be merged. What is your take on this @PeterMatula ?

Reply:
Thanks, I looked in the docs by Mac OS X and I found the parameter `-l`.
I will change it.

Reply:
But I found a package by `brew` the `gnu-time`, this will give the same output. 
Which would be better to integrate?